docs/content/manual/_incoming/1831_Node_disk_manager_should_prevent_too_many_concurrent_disk_formatting_occur_within_a_short_period.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Node disk manager should prevent too many concurrent disk formatting occur within a short period
+category: console
+tags: dashboard, p2, functional
+---
+Ref: https://github.com/harvester/harvester/issues/1831
+
+
+### Criteria
+- [x] exceed the maximum, there should have requeue devices which equals the exceeds
+- [x] hit the maximum, there should not have requeue devices
+- [x] less than maximum, there should not have requeue devices
+
+![image](https://user-images.githubusercontent.com/5169694/177324553-3b4800b2-9db9-45ec-a3cf-a630acb384cf.png)
+
+
+### Verify Steps:
+1. Install Harvester with any node having at least 6 additional disks
+1. Login to console and execute command to update log level to `debug` and `max-concurrent-ops` to `1` (On KVM environment, we have to set to `1` to make sure the _requeuing_ will happen.)
```

Comment:
Would we be able to link to a document or explain how to update the Harvester log level to `debug`?

Reply:
The following command shows how to set `debug` for it.